VideoAPI/VideoApi.IntegrationTests/Database/Queries/GetExpiredAudiorecordingClosedConferencesQueryTests.cs

```diff
@@ -0,0 +1,97 @@
+            _conference1Id = Guid.Empty;
+            _conference2Id = Guid.Empty;
+            _conference3Id = Guid.Empty;
+            _conference4Id = Guid.Empty;
+        }
+
+        [TearDown]
+        public async Task TearDown()
+        {
+            TestContext.WriteLine("Cleaning conferences for GetExpiredAudiorecordingClosedConferencesHandler");
+            await TestDataManager.RemoveConference(_conference1Id);
+            await TestDataManager.RemoveConference(_conference2Id);
+            await TestDataManager.RemoveConference(_conference3Id);
+            await TestDataManager.RemoveConference(_conference4Id);
+        }
+
+        [Test]
+        public async Task Should_return_conferences_with_audiorecording_closed_14_hours_ago()
+        {
+            var utcDate = DateTime.UtcNow;
+            var expired = utcDate.AddHours(-15);
+            var withinTimeLimit = utcDate.AddHours(-10);
+            var beyondTimeLimit = utcDate.AddHours(10);
+
+            var conference1 = new ConferenceBuilder(true, scheduledDateTime: expired)
+                .WithParticipant(UserRole.Representative, "Defendant")
+                .WithParticipant(UserRole.Judge, null)
+                .WithConferenceStatus(ConferenceState.Closed)
+                .WithAudioRecordingRequired(true)
+                .Build();
+            _conference1Id = conference1.Id;
+
+            var conference2 = new ConferenceBuilder(true, scheduledDateTime: withinTimeLimit)
+                .WithParticipant(UserRole.Representative, "Defendant")
+                .WithParticipant(UserRole.Judge, null)
+                .WithConferenceStatus(ConferenceState.InSession)
+                .Build();
+            _conference2Id = conference2.Id;
+
+            var conference3 = new ConferenceBuilder(true, scheduledDateTime: beyondTimeLimit)
+                .WithParticipant(UserRole.Representative, "Defendant")
+                .WithParticipant(UserRole.Judge, null)
+                .WithConferenceStatus(ConferenceState.Paused)
+                .Build();
+            _conference3Id = conference3.Id;
+
+            var conference4 = new ConferenceBuilder(true, scheduledDateTime: expired)
+                .WithParticipant(UserRole.Representative, "Defendant")
+                .WithParticipant(UserRole.Judge, null)
+                .WithConferenceStatus(ConferenceState.Paused)
+                .Build();
+            _conference4Id = conference4.Id;
+
+            await TestDataManager.SeedConference(conference1);
+            await TestDataManager.SeedConference(conference2);
+            await TestDataManager.SeedConference(conference3);
+            await TestDataManager.SeedConference(conference4);
+
+            var conferences = await _handler.Handle(new GetExpiredAudiorecordingClosedConferencesQuery());
+            var confIds = conferences.Select(x => x.Id).ToList();
+            confIds.Should().Contain(conference1.Id);
```

Comment:
you are not actually asserting that the conferences that have audio recording are retrieved? You are getting a response with a list of conferences, getting their ids, then checking that those conferences contain those ids which will always be true?

Reply:
No, I have 4 conferences and know the Ids, 1st and 4th should be with audio recording and status > 0 and expired, one is closed, second is paused.  So I have checked that my list has 1st and 4th, and not contained 2nd and 3rd.